examples/Serving-An-XGboost-Model-With-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>In the title, it should be Serving an XGBoost Model with Merlin Systems -&gt; XG*B*oost. I think An and With should be lowercase letters.</p><p> </p><p>deployt -&gt; deploy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Fixed! Will capitalize titles like you advise going forward